cobalt/build/testing/targets/linux-x64x11/test_targets.json

```diff
@@ -27,7 +27,7 @@
     "services/service_manager/tests:service_manager_unittests",
     "skia:skia_unittests",
     "sql:sql_unittests",
-    "starboard:starboard_unittests",
+    "starboard:starboard_unittests_wrapper",
```

Comment:
Putting a non-test target here isn't going to work. The infra expects a runtime deps file to be created for each of these targets and currently that's only done for test targets.

What are you trying to achieve by putting the group target here?

Reply:
@andrewsavage1 knows better the intention, but IIUC we want to have a trampoline GN target to make sure we build with the `starboard_toolchain`. What would be a good way to codify that?

Reply:
Let's wait on this until #4464 lands

Reply:
#4464 has landed, let's rerun the bots.

Reply:
Bots look green now - @andrewsavage1 can you review?